docs/tutorials.md

```diff
@@ -46,6 +46,14 @@
     "editor": "py",
     "cardType": "tutorial"
   }]
+}, {
+  "name": "Winter",
+  "description": "Help the snow people catch snowflakes!",
+  "url": "/tutorials/winter",
+  "cardType": "tutorial",
+  "imageUrl": "/static/tutorials/winter/winter.png",
+  "largeImageUrl": "/static/tutorials/winter/winter.gif",
+  }]
```

Comment:
i think this line (the `}]` characters) should be removed, it's leftover from the "otherActions" array that the other tutorial cards have

Reply:
Fixed!